assets/js/custom/Translation.js

```diff
@@ -1,7 +1,121 @@
 /* eslint-env jquery */
 
 // eslint-disable-next-line no-unused-vars
-function Translation (hasDescription, hasCredit) {
+function Translation (hasDescription, hasCredit, translatedByText) {
+  const providerMap = {
+    itranslate: 'iTranslate'
+  }
+  let languageMap = {}
+  let targetLanguage = document.documentElement.lang

```

Comment:
i think we can make the behaviour of cookie taking precedence over document...lang more explicit by move this assignment to the end of `setTargetLanguage`

Reply:
Moved to the setTargetLanguage function